demo/src/source-info.js

```diff
@@ -12,20 +12,19 @@ const tiffInfo = {
   }
 };
 
+// We store all chunks in the same pyramid so they all have the same URL.
+// TODO: Add logic for when pyramids are in separate files; most similar to tiff.
```

Comment:
Please file an issue so this doesn't get lost.

Reply:
done